CHANGELOG.md

```diff
@@ -1,5 +1,76 @@
+  * In some occasions, PUT operations on managedClusters or agentPools see long latency in the operations.  This fixes one of a few scenarios.
+  * PATCH operations were allowed on managedClusters in a non-terminal provisioningState.  This could cause an eTag mismatch and inconsistent results or failures.  PATCH operations will not be block for managedClusters in a non-terminal provisioningState.
+* Behavioral Change
+  * Updates to optimize the kube-reserved eviction thresholds available in 1.28
+  * Update [gatekeeper to v3.13.3](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.13.3) and policy addon 1.2.1
+    * Azure Policy Changes
+      * Introduce warn for policies, available in select upcoming built-in policy experiences
+      * Show an exempt ComplianceReasonCode in the portal for exempt policies.
+  * Cluster network settings can be updated to enable Kubenet -> CNI Overlay migration - available in the [CLI](https://github.com/Azure/azure-cli-extensions/pull/6936)
+  * Updates to apiserver to use max value for memory limit if the suggested value is larger.
+  * Added a check for initialization taints that are added on agent pools, only allowing CriticalAddonsOnly taints or soft taints
+  * Enable `UnauthenticatedHTTP2DOSMitigation`` for api server to address potential HTTP2 DOS attack.  For the following versions:
+    * 1.28.3 or up till 1.29.0
+    * 1.27.7 or up till 1.28.0
+    * 1.26.10 or up till 1.27.0
+    * 1.25.15 or up till 1.26.0
+  * Preview feature registration is no longer required for the KEDA addon
+* Component Updates
+  * Update the aks-app-routing-operator to [version 0.0.7](https://github.com/Azure/aks-app-routing-operator/blob/main/CHANGELOG.md#007---2023-11-04) which includes notable changes in [version 0.0.6](https://github.com/Azure/aks-app-routing-operator/blob/main/CHANGELOG.md#006---2023-10-27).
+    * This update has 3 CVE fixes for the nginx ingress controller.
+      * [CVE-2023-5044](https://nvd.nist.gov/vuln/detail/CVE-2023-5044)
+      * [CVE-2022-4886](https://nvd.nist.gov/vuln/detail/CVE-2023-4886)
+      * [CVE-2023-5043](https://nvd.nist.gov/vuln/detail/CVE-2023-5043)
+    * The following changes are also included:
+      * The AJP protocol is no longer supported.
+      * The `whitelist-source-range` annotation has been renamed to `allowlist-source-range`. Both are currently supported but it is recommended to move to the new annotation `allowlist-source-range`.
+    * The `custom-http-errors` annotation now only supports errors between 400 and 599.
+  * Azure Monitor Metrics [November release](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-11-03-2023)
```

Comment:
Update to version XYZ? The image users can see should have a semver version?

Reply:
Not sure what you're asking here - you want something other than a link to the relnotes?

Reply:
The link is fine, the name of the update should contain the semver version to the image the customer can see in their cluster instead of "November"

Reply:
where do we find semver?

Reply:
added